stetho/src/main/java/com/facebook/stetho/inspector/elements/android/AccessibilityNodeInfoWrapper.java

```diff
@@ -100,4 +108,165 @@ public static String getIgnoredReasons(AccessibilityNodeInfoCompat node, View vi
+        return
+            "View is a direct child of a scrollable container and has non-actionable " +
+            "descendants with descriptions.";
+      }
+    }
+
+    if (hasText) {
+      return "View has a description and is not actionable, but has no actionable ancestor.";
+    }
+
+    return null;
+  }
+
+  @Nullable
+  public static String getActions(AccessibilityNodeInfoCompat node) {
+    List<CharSequence> actionLabels = new ArrayList<>();
+
+    for (AccessibilityActionCompat action : node.getActionList()) {
+      switch (action.getId()) {
+        case AccessibilityNodeInfoCompat.ACTION_FOCUS:
+          actionLabels.add("focus");
+          break;
+        case AccessibilityNodeInfoCompat.ACTION_CLEAR_FOCUS:
+          actionLabels.add("clear-focus");
+          break;
+        case AccessibilityNodeInfoCompat.ACTION_SELECT:
+          actionLabels.add("select");
+          break;
+        case AccessibilityNodeInfoCompat.ACTION_CLEAR_SELECTION:
+          actionLabels.add("clear-selection");
+          break;
+        case AccessibilityNodeInfoCompat.ACTION_CLICK:
+          actionLabels.add("click");
+          break;
+        case AccessibilityNodeInfoCompat.ACTION_LONG_CLICK:
+          actionLabels.add("long-click");
+          break;
+        case AccessibilityNodeInfoCompat.ACTION_ACCESSIBILITY_FOCUS:
+          actionLabels.add("accessibility-focus");
+          break;
+        case AccessibilityNodeInfoCompat.ACTION_CLEAR_ACCESSIBILITY_FOCUS:
+          actionLabels.add("clear-accessibility-focus");
+          break;
+        case AccessibilityNodeInfoCompat.ACTION_NEXT_AT_MOVEMENT_GRANULARITY:
+          actionLabels.add("next-at-movement-granularity");
+          break;
+        case AccessibilityNodeInfoCompat.ACTION_PREVIOUS_AT_MOVEMENT_GRANULARITY:
+          actionLabels.add("previous-at-movement-granularity");
+          break;
+        case AccessibilityNodeInfoCompat.ACTION_NEXT_HTML_ELEMENT:
+          actionLabels.add("next-html-element");
+          break;
+        case AccessibilityNodeInfoCompat.ACTION_PREVIOUS_HTML_ELEMENT:
+          actionLabels.add("previous-html-element");
+          break;
+        case AccessibilityNodeInfoCompat.ACTION_SCROLL_FORWARD:
+          actionLabels.add("scroll-forward");
+          break;
+        case AccessibilityNodeInfoCompat.ACTION_SCROLL_BACKWARD:
+          actionLabels.add("scroll-backward");
+          break;
+        case AccessibilityNodeInfoCompat.ACTION_CUT:
+          actionLabels.add("cut");
+          break;
+        case AccessibilityNodeInfoCompat.ACTION_COPY:
+          actionLabels.add("copy");
+          break;
+        case AccessibilityNodeInfoCompat.ACTION_PASTE:
+          actionLabels.add("paste");
+          break;
+        case AccessibilityNodeInfoCompat.ACTION_SET_SELECTION:
+          actionLabels.add("set-selection");
+          break;
+        default:
+          if (action.getLabel() != null) {
```

Comment:
not sure if `getLabel()` allocates and returns a new `String` each time. If so, it's important to call `getLabel()` only once. In principle I'd do that anyway as a defensive measure.